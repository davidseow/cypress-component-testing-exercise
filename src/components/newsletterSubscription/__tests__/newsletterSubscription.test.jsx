import { render } from "@testing-library/react";
import NewsletterSubscription from "../newsletterSubscription";

describe("<NewsletterSubscription/>", () => {
  test("should render", () => {
    const { asFragment } = render(<NewsletterSubscription />);

    expect(asFragment()).toMatchSnapshot();
  });
});
