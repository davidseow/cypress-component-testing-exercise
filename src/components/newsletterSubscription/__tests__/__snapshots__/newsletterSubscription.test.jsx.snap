// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NewsletterSubscription/> should render 1`] = `
<DocumentFragment>
  <div
    id="newsletter"
  >
    Newsletter
  </div>
</DocumentFragment>
`;
